index.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const { DEFAULT_LENGTH } = require('./utils/constants');
+const getPixelCoordinates = require('./src/get-pixel-coordinates');
+const getSiblingTilesAndOffsets = require('./src/get-sibling-tiles-and-offsets');
+const getAllTiles = require('./src/get-all-static-tiles');
+const stitchTiles = require('./src/stitch-tiles');
+
+
+function makeGetQuilt(coordinates, zoom, length = DEFAULT_LENGTH, decode = true) {
+  return (style) => {
```

Comment:
Curious why this isn't the other way around. Wouldn't it be more common to initialize with a style/token and then call it repeatedly with `coordinates`, and `zoom`?

Reply:
Currently, I'm just focused on keeping this as it's currently being used downstream which follows this pattern. But I'm not opposed to inverting the `make/get` relationship, as it does make more sense the way you've outlined.

Reply:
Obvs your call here but is this something you would like to do instead? It's a pretty breaking change if it's introduced later on.

Reply:
I did a quick review of the downstream implementations and I think shifting this structure would cause us some non-trivial refactoring for this use case. My gut reaction is to defer as much downstream refactoring as possible, and accept this pattern as is for now.

If we decide to use this library in more places in the future (and find ourselves frustrated by the pattern), then the breaking change would be more worth the effort.